### ISSUE TITLE
Test fails since 'lazy_load_hdus' keyword was not found 

### ISSUE THREAD

**Issue by olebole** [spacetelescope/stsci.tools#48]
When running the tests with astropy 2.0.2, I got the following failure:
```
ERROR: Test the PrimaryHDU constructor in both the fits and stpyfits
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/usr/lib/python2.7/dist-packages/nose/case.py", line 197, in runTest
    self.test(*self.arg)
  File "/build/stsci.tools-3.4.11/.pybuild/pythonX.Y_2.7/build/stsci/tools/tests/testStpyfits.py", line 358, in testPrimaryHDUConstructor
    overwrite=True)
  File "/build/stsci.tools-3.4.11/.pybuild/pythonX.Y_2.7/build/stsci/tools/stpyfits.py", line 55, in wrapped_with_stpyfits
    retval = func(*args, **kwargs)
  File "/usr/lib/python2.7/dist-packages/astropy/utils/decorators.py", line 507, in wrapper
    return function(*args, **kwargs)
TypeError: writeto() got an unexpected keyword argument 'lazy_load_hdus'
```

**Comment by pllim** [comment]
Is this related to #38, @jhunkeler ?

**Comment by jhunkeler** [comment]
@pllim  `TypeError: writeto() got an unexpected keyword argument 'lazy_load_hdus'` is getting raised by the `lazy_load_hdus` workaround implemented here:
https://github.com/spacetelescope/stsci.tools/blob/master/lib/stsci/tools/stpyfits.py#L50-L55

Hmm.

**Comment by pllim** [comment]
@jhunkeler , yes, and I tagged you according to `git blame`. :sweat_smile:

**Comment by pllim** [comment]
Fixed by #61 (more like a hack but tests are not complaining anymore)
